### PR TITLE
Git instructions

### PR DESCRIPTION
**Pull request by BartoszAndrzejczyk**

### CODE REVIEW

File: git.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+### commit
+
+`git commit` saves changes
+
+Example:
+```console
+$ git commit -m "Enter here a message"
+```
+### status
+`git status`  Shows the state of working directory.
```
Comment (rojarand) [inline git.md]: There is an unneeded space before 'Show'.  Make it as a sentence: `git status` shows the state ...

File: git.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+```console
+$ git status
+```
+### push
+`git push` It lets you move a local branch to another repository.
```
Comment (rojarand) [inline git.md]: Make it as a sentence: `git push` moves a local ...

File: git.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+```console
+$ git push origin main
+```
+### add
+`git add` Moves changes from the working directory to the staging area.
```
Comment (rojarand) [inline git.md]: Make it as a sentence: `git add` moves changes. Move it above `git push`

File: git.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+```console
+$ git add FROG.md
+```
+### diff 
+`git diff` Diff command is used in git to track the difference between the changes made on a file.
```
Comment (rojarand) [inline git.md]: Make it as a sentence: `git diff` shows difference ...

File: git.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+```console
+$ git diff
+```
+### merge
+`git merge` It's a way to integrate changes from divergent branches.
```
Comment (rojarand) [inline git.md]: Make it as a sentence above

File: git.md (hunk 1 above)
Comment (BartoszAndrzejczyk) [reply git.md]: Fixed

File: git.md (hunk 2 above)
Comment (BartoszAndrzejczyk) [reply git.md]: Done thank's

File: git.md (hunk 5 above)
Comment (BartoszAndrzejczyk) [reply git.md]: Fixed

File: git.md (hunk 4 above)
Comment (BartoszAndrzejczyk) [reply git.md]: Fixed

File: git.md (hunk 3 above)
Comment (BartoszAndrzejczyk) [reply git.md]: fixed

File: git.md (hunk 5 above)
Comment (BartoszAndrzejczyk) [reply git.md]: Thank's for help

**Review by rojarand — APPROVED** [review]

**Pull request merged**